### ISSUE TITLE
Inaccuracy in guide-ru with checkboxList

### ISSUE THREAD

**Issue by mikbox74** [yiisoft/yii2#9097]
https://github.com/yiisoft/yii2/blob/master/docs/guide-ru/input-forms.md

```php
// поддерживает выбор нескольких значений:
echo $form
    ->field($model, 'items[]')
    ->checkboxList(['a' => 'Item A', 'b' => 'Item B', 'c' => 'Item C']);
```

When field name used width [], automatic population from model not works.
The field method automatically adds [] if it is necessary.

**Comment by lynicidn** [comment]
may be need `[]items`

**Comment by SilverFire** [comment]
Duplicates https://github.com/yiisoft/yii2/issues/11973, to be fixed in 2.1. See https://github.com/yiisoft/yii2/issues/12407
